concent_api/utils/helpers.py

```diff
@@ -116,15 +116,18 @@ def get_storage_file_path(category, subtask_id, task_id):
     return f'blender/{category}/{task_id}/{task_id}.{subtask_id}.zip'
 
 
-def get_storage_result_file_path(subtask_id, task_id):
+def get_storage_result_file_path(subtask_id=None, task_id=None):
+    assert subtask_id is not None and task_id is not None
```

Comment:
I think they're fine, but `get_storage_file_path()` should have them too.

The thing is that calling `get_storage_file_path()` with `None` does not make sense (it will give you a bad path) so that function must indicate that with an `assert`. Otherwise `None` is considered valid input and the function has a bug (it does not handle it correctly). So as a result calling `get_storage_result_file_path()` with `None` does not make sense either it should indicate that too.

Having said that, since `get_storage_result_file_path()` is an extremely simple one-liner, I'd probably put the assertion only in `get_storage_file_path()`.

Reply:
Done